version/versions.yaml

```diff
@@ -1,4 +1,6 @@
 - operatorVersion: 1.3.0
   supportedIstioVersions: 1.3.0
   recommendedIstioVersions: 1.3.0
```

Comment:
Since we're updating this, could you please add an entry for 1.3.1 too?

Reply:
done